documentation/SA1026.md

```diff
@@ -0,0 +1,39 @@
+﻿# SA1026
```

Comment:
:bulb: When adding a documentation file, the `HelpLink` URI for the analyzer needs to be updated to reflect the new location of the documentation.

:bulb: When adding a documentation file, the file should be added to the **documentation** node in Solution Explorer.